packages/market/client/settings/utils.ts

```diff
@@ -45,7 +45,7 @@ function getEnvInfo(name: string) {
   const services = new Set<string>()
 
   // check peer dependencies
-  for (const name in local.peerDependencies ?? {}) {
+  for (const name in Object.keys(local.peerDependencies ?? {})) {
```

Comment:
[`Object.keys()`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) would return an `array` of all the enumerable keys of the current object, then if you `for-in` the `array` you would get a number instead of the key name.
Although I don't know what problem you met. It should be `for-of`, no?

Reply:
Yes, I must have made a mistake here.
I opened this pr because I had a bug in my console that would show optional dependencies as required dependencies.

I have figured out the key problem and will push another commit to fix this soon.

Reply:
I think this commit should fix it :)